src/components/PeopleList.tsx

```diff
@@ -0,0 +1,88 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  listOfPeople: Person[];
+};
+
+export const PeopleList: React.FC<Props> = ({ listOfPeople }) => {
+  const { selectedSlug = 0 } = useParams();
+
+  const list = listOfPeople.map(personage => {
+    const mother = listOfPeople.find(she => she.name === personage.motherName);
+    const father = listOfPeople.find(he => he.name === personage.fatherName);
```

Comment:
Better to use more meaningful names

```suggestion
    const mother = listOfPeople.find(person => person.name === personAge.motherName);
    const father = listOfPeople.find(person => person.name === personAge.fatherName);
```